modules/sqs/variables.tf

```diff
@@ -24,6 +24,16 @@ variable "metadata_queue_message_retention_time_seconds" {
   description = "The number of seconds metadata-queue fifo SQS retains a message in seconds. Maximum value is 14 days."
 }
 
+variable "orca_reconciliation_lambda_timeout" {
+  type        = number
+  description = "Timeout in number of seconds for ORCA reconciliation lambdas."
+}
+
+variable "s3_inventory_queue_message_retention_time_seconds" {
+  type        = number
+  description = "The number of seconds s3-inventory-queue fifo SQS retains a message in seconds. Maximum value is 14 days."
```

Comment:
Per our discussion on FIFO and how a FIFO DLQ we think will break the issue where processing would stop because order must be maintained, will leave as FIFO for now until we see some real world use/testing.